rero_mef/monitoring/cli.py

```
@@ -165,7 +167,8 @@ def db_connection_counts():
         ).first()
     except Exception as error:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)